akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowWithContextLogSpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2014-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.scaladsl
+
+import akka.event.Logging
+import akka.stream.Attributes.LogLevels
+import akka.stream.testkit.{ StreamSpec, ScriptedTest }
+import akka.stream._
+import akka.testkit.TestProbe
+
+class FlowWithContextLogSpec extends StreamSpec("""
+     akka.loglevel = DEBUG # test verifies logging
+     akka.actor.serialize-messages = off
+     """) with ScriptedTest {
+
+  implicit val mat: Materializer = ActorMaterializer()
+
+  val logProbe = {
+    val p = TestProbe()
+    system.eventStream.subscribe(p.ref, classOf[Logging.LogEvent])
```

Comment:
This method of capturing logs (rather than an EventFilter) may end up failing due to other logs added but okay for now